rules/windows/privilege_escalation_lsa_auth_package.toml

```diff
@@ -0,0 +1,68 @@
+[metadata]
+creation_date = "2021/01/21"
+maturity = "production"
+updated_date = "2021/01/21"
+
+[rule]
+author = ["Elastic"]
+description = """
+Adversaries can use the autostart mechanism provided by the Local Security Authority (LSA) authentication packages for
+privilege escalation or persistence by placing a reference to a binary in the Windows registry. The binary will then be
+executed by SYSTEM when the authentication packages are loaded.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*"]
+language = "eql"
+license = "Elastic License"
+name = "Potential LSA Authentication Package Abuse"
+risk_score = 47
+rule_id = "e9abe69b-1deb-4e19-ac4a-5d5ac00f72eb"
+severity = "medium"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Privilege Escalation"]
+type = "eql"
+
+query = '''
+registry where event.type : "change" and
+  registry.path : "HKLM\\SYSTEM\\*ControlSet*\\Control\\Lsa\\Authentication Packages" and
+  /* exclude SYSTEM SID - look for changes by non-SYSTEM user */
+  not user.id:"S-1-5-18"
```

Comment:
```suggestion
  not user.id : "S-1-5-18"
```

Should this spacing be adjusted for consistency?

Reply:
yes it shouls